Automattic/wp-calypso
# Happychat: Middleware refactor (Transcripts)

Replacing previous attempts in #11381 and #11604, this lays the groundwork to iteratively refactor parts of the actions into middleware, starting with the transcript requests.

### How to test

1. Make yourself available as an operator in happychat staging.
2. Go to http://calypso.localhost:3000/help/contact and start a chat.
3. Start a chat, send messages from both user and operator
4. End the chat
5. Refresh calypso and start a new chat, confirming that the previous chat's transcripts appear in the Calypso chat window
